automatic/wps-office-free/update.ps1

```diff
@@ -0,0 +1,48 @@
+  import-module au
+  
+  $releases = 'https://www.wps.com/office-free'
+  
+  function global:au_SearchReplace {
+      @{
+          ".\tools\chocolateyInstall.ps1" = @{
+		      "(^[$]version\s*=\s*)('.*')"= "`$1'$($Latest.Version)'"
+              "(^\s*packageName\s*=\s*)('.*')"= "`$1'$($Latest.PackageName)'"
+              "(^\s*url\s*=\s*)('.*')" = "`$1'$($Latest.URL32)'"
+              "(^\s*softwareName\s*=\s*)('.*')"= "`$1'$($Latest.SoftwareName)'"
+              "(^\s*checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+              "(^\s*checksumType\s*=\s*)('.*')" = "`$1'$($Latest.ChecksumType32)'"
+          }
+          "wps-office-free.nuspec" = @{
+              "(\<description\>).*(\<\/description\>)" = "`${1}$($Latest.Description)`$2"
```

Comment:
this replacement won't/shouldn't work since the description is multiline.
If you really need to replace something in the description, only replace what is needed.
For instance the year in `Windows WPS Office 2016 Free`, If you decide to do that, do the same for the title element then.